### DISCUSSION TITLE
CalendarPicker / CalendarPickerView language selection

### DISCUSSION THREAD

**Discussion by jnorvell** [ant-design/ant-design-mobile#6630] [Q&A] [answered]
People are using these components with English language, how to configure that?   Thanks

**Comment by yezhonghu0503** [comment, marked as answer]
this is simple demo : [language selection](https://codesandbox.io/p/sandbox/fancy-platform-5py9n7?file=%2Fapp.tsx%3A27%2C36&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clwhab2wd0007356juhs5t5x6%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clwhab2wc0003356jfc6s80f3%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clwhab2wc0004356jfrzypjnc%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clwhab2wc0006356j69woz4n0%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clwhab2wc0003356jfc6s80f3%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clwhab2wc0002356jesgwutdb%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fapp.tsx%2522%252C%2522state%2522%253A%2522IDLE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A27%252C%2522startColumn%2522%253A36%252C%2522endLineNumber%2522%253A27%252C%2522endColumn%2522%253A36%257D%255D%257D%255D%252C%2522id%2522%253A%2522clwhab2wc0003356jfc6s80f3%2522%252C%2522activeTabId%2522%253A%2522clwhab2wc0002356jesgwutdb%2522%257D%252C%2522clwhab2wc0006356j69woz4n0%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clwhab2wc0005356j5jubtbbh%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clwhab2wc0006356j69woz4n0%2522%252C%2522activeTabId%2522%253A%2522clwhab2wc0005356j5jubtbbh%2522%257D%252C%2522clwhab2wc0004356jfrzypjnc%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clwhab2wc0004356jfrzypjnc%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

Global usage and more ways to use it: https://mobile.ant.design/zh/guide/i18n

**Reply by jnorvell** [reply]
Thank you!
